market/utils/web-tester/README.md

```diff
@@ -0,0 +1,13 @@
+# Web Tester
+
+The `web-tester` artifact provides you a API which helps you test your HTML-Dialogs.
+With this API it is easy to setup your test environment and send requests
+against your [Axon Ivy Engine](/download).
```

Comment:
Use absolute links. We may move market to another subdomain etc. market.axonivy.com